structures/rels.js

```diff
@@ -3,15 +3,23 @@ const Url = s.define('Url', require('is-url'))
 const Instance = process.env.NODE_APP_INSTANCE || 'DEMO'
 const Domain = process.env.DNSDOMAIN || '.local'
 
+const toZoulouDate = value => {
+  return value.slice(-1) !== 'Z' ? value + 'Z' : value
+}
+
+const ZoulouDate = require('./lib').zouloudate(s)
+
 const Rel = s.defaulted(
   s.object({
     key: s.size(s.string(), 2, 32),
     name: s.optional(s.size(s.string(), 2, 64)),
     url: s.optional(Url),
```

Comment:
Add `type` (`public` | `private`)

Reply:
I have added "visible" to be conssisten with the auction argument.